src/components/tutorial-meta/components/badges/badges.module.css

```diff
@@ -0,0 +1,22 @@
+ul.list {
+  --badge-spacing: 8px;
+
+  margin-bottom: calc(var(--spacing-lrg) - var(--badge-spacing));
+  padding-left: 0;
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+
+  @media (--medium-up) {
+    margin-bottom: calc(var(--spacing-xl) - var(--badge-spacing));
+  }
+
+  & li:not(:last-child) {
+    margin-bottom: var(--badge-spacing);
+  }
+}
+
+.beta {
+  background-color: var(--token-color-highlight-background-faint);
+  color: var(--token-color-palette-purple-300) !important;
```

Comment:
There are some default styles on[ dev dot content root](https://github.com/hashicorp/dev-portal/blob/main/src/components/dev-dot-content/dev-dot-content.module.css) that were interferring with these badge styles. I think that these content styles should only apply to the actual markdown content area 🤷 For now, this is my workaround.

Reply:
Is it line 72 in that file that's interfering?

https://github.com/hashicorp/dev-portal/blob/main/src/components/dev-dot-content/dev-dot-content.module.css#L72

Reply:
@ashleemboyer yes! thats the file. There were some other instances where I had to wiggle with specificity because of these root content styles also.

Reply:
@zchsh I think that would be a great shift. Also I noticed that these styles are applied in the sidebar / sidecar layout level, and maybe it would be better for these styles to apply to a specific `Content` component that just renders mdx content? Since the sidebar / sidecar layout will likely render other non-mdx content components in various views. Just a thought.

Reply:
☝️ I dig that - we may have abstracted the use of content styles in sidebar-sidecar layout a little early, I think mostly for the sake of less repetition in docs views (each of the many page files would all have to import said `Content` component and then use it in the exact same way, I think... which is fine, just repetitive). Ashlee's work in https://github.com/hashicorp/dev-portal/pull/214 seems like it'd DRY up the docs view situation, which feels like it could make the `Content`-outside-of-sidebar-sidecar approach easier to reach for 👍

Reply:
@kendallstrautman the linked file is from the new `DevDotContent` component 😊

Reply:
Yes. Another possibility we could do is add a `headerSlot` prop to `SidebarSidecarLayout`, and render that as the sibling before the `DevDotContent` that wraps `children`.